Metagenomics/Remove_human_reads_from_raw_data/Workflow_Documentation/NF_MGRemoveHumanReads-A/workflow_code/README.md

```diff
@@ -0,0 +1,154 @@
+# NF_MGRemoveHumanReads-A Workflow Information and Usage Instructions
+
+
+## General workflow info
+The current pipeline for how GeneLab identifies and removes human DNA in Illumina metagenomics sequencing data (MGRemoveHumanReads), [GL-DPPD-7105-A.md](../../Pipeline_GL-DPPD-7105_Versions/GL-DPPD-7105-A.md), is implemented as a [NextFlow](https://www.nextflow.io/docs/stable/index.html) DSL2 workflow and utilizes [Singularity](https://docs.sylabs.io/guides/3.10/user-guide/introduction.html) run all tools in containers. This workflow (NF_MGRemoveHumanReads-A) is run using the command line interface (CLI) of any unix-based system. The workflow can be used even if you are unfamiliar with NextFlow and Singularity, but if you want to learn more about those, [this NextFlow tutorial](https://training.nextflow.io/basic_training/) within [NextFlow's documentation](https://www.nextflow.io/docs/stable/index.html) is a good place to start for that.
+
+## Utilizing the workflow
+
+1. [Install conda, mamba, and `genelab-utils` package](#1-install-conda-mamba-and-genelab-utils-package) 
+2. [Install NextFlow and Singularity](#2-install-NextFlow-Singularity)
+   2a. [Install Nextflow](#2a-install-nextflow)  
+   2b. [Install Singularity](#2b-install-singularity)
+3. [Download the workflow template files](#3-download-the-workflow-template-files)  
+4. [Modify the variables in the Remove_Human_Reads.config file](#4-modify-the-variables-in-the-config-file)  
+5. [Run the workflow](#5-run-the-workflow)  
+
+
+<br>
+
+---
+
+### 1. Install conda, mamba, and `genelab-utils` package
+We recommend installing a Miniconda, Python3 version appropriate for your system, as exemplified in [the above link](https://astrobiomike.github.io/unix/conda-intro#getting-and-installing-conda).  
```

Comment:
The text "the above link" isn't referring to anything above and should be updated to be more specific about where its pointing.
```suggestion
We recommend installing a Miniconda, Python3 version appropriate for your system, as exemplified in [the Happy Belly Bioinformatics conda tutorial](https://astrobiomike.github.io/unix/conda-intro#getting-and-installing-conda).  
```